packages/lambda-tiler/src/__test__/xyz.test.ts

```diff
@@ -198,6 +198,46 @@ o.spec('LambdaXyz', () => {
         });
     });
 
+    o.spec('tileJson', () => {
+        const origPublicUrlBase = process.env[Env.PublicUrlBase];
+        o.after(() => {
+            process.env[Env.PublicUrlBase] = origPublicUrlBase;
+        });
+
+        o('should 304 if a json is not modified', async () => {
+            const key = 'fYC3MHM69e9U9ltWQLry45/G1wMjTeauij8L2HUdVk4=';
+            const request = mockRequest('/v1/tiles/tile.json', 'get', { 'if-none-match': key });
+
+            const res = await handleRequest(request);
+            if (res.status === 200) {
+                o(res.header('eTaG')).equals(key); // this line is useful for discovering the new etag
+                return;
```

Comment:
do we want to throw if the status is 200?

Reply:
This is used to discovering the if the etag changes. I don't think tilejson is immutable. We still want to update the layers.

Reply:
does the test fail if `status == 200`